public/app/features/logs/components/LogDetails.tsx

```diff
@@ -70,7 +70,15 @@ class UnThemedLogDetails extends PureComponent<Props> {
     const labels = row.labels ? row.labels : {};
     const labelsAvailable = Object.keys(labels).length > 0;
     const fieldsAndLinks = getAllFields(row, getFieldLinks);
-    const links = fieldsAndLinks.filter((f) => f.links?.length).sort();
+    const links = fieldsAndLinks
+      .filter((f) => {
+        if (f.links?.length) {
+          return f.links.filter((link) => link.wasQueryInterpolated).length;
```

Comment:
I don't think we want to show the linked lines unless something was changed about the query. Otherwise if there's a correlation, it will show in every line even if the query still contains the template variable

Reply:
I think we could simply not return the link instead of having an extra prop.